blog/_posts/2019-11-13-find-usages.adoc

```diff
@@ -157,7 +157,7 @@ These restrictions can originate from the language semantics: it doesn't make se
 They also can originate from the user: it's often convenient to exclude tests from search results, for example.
 
 The second trick is about implementing warnings for unused declarations effectively.
-This is a case where a top-down approach is generally better, as an IDE needs to process every declaration, and that would be slow with top-down approach.
+This is a case where a top-down approach is generally better, as an IDE needs to process every declaration, and that would be slow with bottom-up approach.
```

Comment:
This should have said "a bottom-up approach", but I've fixed it manually :smile:.

Reply:
Oh, I missed this one. Thanks!